### ISSUE TITLE
Rule to enforce nesting of elements

### ISSUE THREAD

**Issue by hisuwh** [kristerkari/stylelint-scss#197]
I want stylelint to complain if I do this:

```css
.something .something-else {
    width: 100px;
}
```

And be happy if I do this:
```css
.something {
    .something-else {
        width: 100px;
    }
}
```

Much like sass-lint's [force-element-nesting](https://github.com/sasstools/sass-lint/blob/master/docs/rules/force-element-nesting.md)

**Comment by kristerkari** [comment]
Thanks! That sounds like a good rule idea

**Comment by kristerkari** [comment]
This now available with ` selector-nest-combinators` rule in v3.5.0

I also opened a new issue to add more rules to enforce nesting:
https://github.com/kristerkari/stylelint-scss/issues/299
